app/Actions/Diagnostics/Checks/IniSettingsCheck.php

```diff
@@ -77,5 +73,10 @@ public function check(array &$errors): void
 			$errors[]
 				= 'Warning: exec function has been disabled. You may experience some error 500, please report them to us.';
 		}
+
+		if (empty(ini_get('user_agent'))) {
+			$errors[]
+				= 'Warning: php_agent is not set. You may experiences problems when importing images via URL.';
```

Comment:
Propsol for bettet message: "Warning: user_agent for PHP is not set. .....".
The configuration option `user_agent` should be named correctly.

Reply:
```suggestion
				= 'Warning: php_agent for PHP is not set. You may experiences problems when importing images via URL.';
```

Reply:
```
```suggestion
```
is the way to go @nagmat84  ;)

Reply:
```suggestion
				= 'Warning: user_agent for PHP is not set. You may experiences problems when importing images via URL.';
```

I added my counter-suggestion ;-) I wanted to say that the name of the config option is `user_agent` not `php_agent`.